src/lib/email.ts

```diff
@@ -31,13 +33,18 @@ export class EmailAPI {
   }): Promise<SendEmailResponse> {
     const { recipients, cc, bcc, sender, contents } = options;
     if (!recipients || !sender || !contents) {
-      throw new Error(
+      throw new JunoValidationError(
         'Parameter recipients or sender or content cannot be null'
       );
     }
-    if (recipients.length === 0 || contents.length === 0) {
-      throw new Error(
-        'Parameter recipients or content cannot be an empty array'
+
+    if (recipients.length === 0 && cc?.length === 0 && bcc?.length === 0) {
```

Comment:
Recipients can be length 0 (it was previously assumed it couldn't). This will likely be something we need to change with the schema as well considering recipients is not an optional field in it